packages/postcss-discard-unused/src/__tests__/index.js

```diff
@@ -75,21 +75,27 @@ test(
 test(
     'should remove unused fonts & keep used fonts',
     processCSS,
-    '@font-face {font-family:"Does Not Exist";src:url("fonts/does-not-exist.ttf") format("truetype")}@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font-family:"Does Exist",Helvetica,Arial,sans-serif}',
-    '@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font-family:"Does Exist",Helvetica,Arial,sans-serif}'
+    '@font-face {font-family:"Does Not Exist";src:url("fonts/does-not-exist.ttf") format("truetype")}@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font-family: "Does Exist", Helvetica, Arial, sans-serif}',
+    '@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font-family: "Does Exist", Helvetica, Arial, sans-serif}'
 );
 
 test(
     'should work with the font shorthand',
     passthroughCSS,
-    '@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font: 10px/1.5 "Does Exist",Helvetica,Arial,sans-serif}'
+    '@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font: 10px/1.5 "Does Exist", Helvetica, Arial, sans-serif}'
 );
 
 test(
-    'should not be responsible for normalising fonts',
+    'should work with mixed quote styles',
+    passthroughCSS,
+    '@font-face {font-family:"Does Exist";src:url("fonts/does-exist.ttf") format("truetype")}@font-face {font-family:\'DoesExist\';src:url("fonts/does-exist.ttf") format("truetype")}body{font-family: Does Exist; font: 10px DoesExist}'
+);
+
+test(
+    "should handle font properties that don't include a font-family",
     processCSS,
-    '@font-face {font-family:"Does Exist";src:url("fonts/does-exist.ttf") format("truetype")}body{font-family:Does Exist}',
-    'body{font-family:Does Exist}'
+    '@font-face {font-family:"Does Exist";src: url("fonts/does-exist.ttf") format("truetype")}body{font: italic bold 10px/1.5}',
+    'body{font: italic bold 10px/1.5}'
```

Comment:
Don't remove existing tests, modify their output value and/or add new tests

Reply:
Which tests are you referring to? The “should not be responsible for normalising fonts” test was asserting the exact opposite of what this change is doing, it is replaced by the “should work with mixed quote styles” test, which has the same input with some added css and the new output. I added spaces to some tests, because none of the existing tests included spaces. I can reverse the space additions and add a new test to call out handling spaces, if that helps.

Reply:
Yep, better add new tests :+1:

Reply:
Ok, I've reversed the changes to the tests and committed new tests.